scanner/updater/export.go

```diff
@@ -46,23 +46,20 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 	if err != nil {
 		return fmt.Errorf("initializing: manual: %w", err)
 	}
-	bundles["rhel"], err = rhelOpts(ctx)
-	if err != nil {
-		return fmt.Errorf("initializing updater: rhel: %w", err)
-	}
 	bundles["nvd"] = nvdOpts()
 
 	// ClairCore updaters.
 	for _, uSet := range []string{
-		"oracle",
-		"photon",
-		"suse",
-		"aws",
 		"alpine",
+		"aws",
 		"debian",
+		"oracle",
+		"osv",
+		"photon",
 		"rhcc",
+		"rhel-vex",
```

Comment:
I am guessing you're keeping the old updater for safety?

Also, by removing it everyone using the dev updater stream will stop seeing updates unless they update to this commit.

Reply:
> I am guessing you're keeping the old updater for safety?

I'm keeping the code for it around in case we have to revert this change. That way it'd be fewer affected files

Though, I think you are referring to using the old updater vs keeping the code around. If referring to actually using the old updater, I wasn't planning on it (removed around lines 49 - 52)

> Also, by removing it everyone using the dev updater stream will stop seeing updates unless they update to this commit.

True, though I'm ok with that as long as nightlies/staging still works properly

Reply:
> I'm keeping the code for it around in case we have to revert this change. That way it'd be fewer affected files

FYI, You need to keep around the pieces in the mapper for a full rollback.

> Also, by removing it everyone using the dev updater stream will stop seeing updates unless they update to this commit.

Nightlies will be OK since they are release builds, and use `v1` bundles, and not `dev`.

Reply:
https://github.com/stackrox/stackrox/pull/12452/commits/0c62e00fdad8fb37830a6f2523c8b1627346ab48